src/views/thread/components/messages.js

```diff
@@ -48,6 +49,7 @@ type Props = {
   data: {
     thread: {
```

Comment:
Let's type this as `import type { ThreadInfoType } from 'shared/graphql/thread/threadInfo'; data: { thread: ThreadInfoType } }`